{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "80adf34d_35b264fc",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 4
      },
      "lineNbr": 0,
      "author": {
        "id": 1305461
      },
      "writtenOn": "2021-07-07T18:34:35Z",
      "side": 1,
      "message": "Thanks, Ching-Kang!",
      "revId": "f134441b44393446e992aa902eb1d89baad870cf",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "173ecdd0_f83d915a",
        "filename": "src/chromiumos/tast/local/bundles/cros/crash/auth_failure.go",
        "patchSetId": 4
      },
      "lineNbr": 93,
      "author": {
        "id": 1305461
      },
      "writtenOn": "2021-07-07T18:34:35Z",
      "side": 1,
      "message": "is there any reason to wait 30 seconds here, rather than the default of 15 seconds? If I understand correctly, we expect that we will *not* see any crash files here, so this call will cause every execution of the test to take 30 seconds longer.",
      "range": {
        "startLine": 93,
        "startChar": 104,
        "endLine": 93,
        "endChar": 134
      },
      "revId": "f134441b44393446e992aa902eb1d89baad870cf",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "e0fa9847_c0547514",
        "filename": "src/chromiumos/tast/local/bundles/cros/crash/auth_failure.go",
        "patchSetId": 4
      },
      "lineNbr": 98,
      "author": {
        "id": 1305461
      },
      "writtenOn": "2021-07-07T18:34:35Z",
      "side": 1,
      "message": "nit: it might be useful to use crash.MoveFilesToOut to save the crash files for debugging purposes in this case.",
      "range": {
        "startLine": 98,
        "startChar": 0,
        "endLine": 98,
        "endChar": 61
      },
      "revId": "f134441b44393446e992aa902eb1d89baad870cf",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "2f213e64_a61bf3c5",
        "filename": "src/chromiumos/tast/local/bundles/cros/crash/auth_failure.go",
        "patchSetId": 4
      },
      "lineNbr": 115,
      "author": {
        "id": 1305461
      },
      "writtenOn": "2021-07-07T18:34:35Z",
      "side": 1,
      "message": "nit: This two calls are repeated from earlier in the test method; can we pull them out into a helper function?",
      "range": {
        "startLine": 105,
        "startChar": 0,
        "endLine": 115,
        "endChar": 0
      },
      "revId": "f134441b44393446e992aa902eb1d89baad870cf",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}